src/styles/calendar.scss

```diff
@@ -0,0 +1,57 @@
+@import url(./_mixin.scss); // why don't work?
+@import url(./variables.scss); // again
+
```

Comment:
Try to import without url and brackets, just using doublequotes. And import mixins and variables into pages, where you are using them.